### PR TITLE
DT-972 changes done for Beta 2 new schema

### PR DESCRIPTION
**Pull request by preetamnpr**

### CODE REVIEW

File: booking/src/main/java/org/dcsa/conformance/standards/booking/checks/BookingChecks.java [hunk 1]
```diff
@@ -297,6 +316,23 @@ private static Consumer<MultiAttributeValidator> allDg(Consumer<MultiAttributeVa
...
+    (mav) -> {
+      mav.submitAllMatching("advancedManifestFilings.*.countryCode");
+      mav.submitAllMatching("documentParties.*.party.taxLegalReferences.*.countryCode");
+
+      // Beta-2 only
+      mav.submitAllMatching("documentParties.shippers.address.countryCode");
+      mav.submitAllMatching("documentParties.consignee.address.countryCode");
+      mav.submitAllMatching("documentParties.endorsee.address.countryCode");
+      mav.submitAllMatching("documentParties.serviceContractOwner.address.UNLocationCode");
+      mav.submitAllMatching("documentParties.carrierBookingOffice.address.UNLocationCode");
```
Comment (nt-gt) [inline booking/src/main/java/org/dcsa/conformance/standards/booking/checks/BookingChecks.java:330]: Not a countryCode attribute

File: booking/src/main/java/org/dcsa/conformance/standards/booking/party/BookingCarrier.java [hunk 2]
```diff
@@ -652,6 +652,7 @@ public ObjectNode asJsonNode() {
       setIfNotNull(notification, "source", source);
       setIfNotNull(notification, "type", computedType());
       notification.put("time", Instant.now().toString());
+      notification.put("subscriptionReference", UUID.randomUUID().toString());
```
Comment (nt-gt) [inline booking/src/main/java/org/dcsa/conformance/standards/booking/party/BookingCarrier.java:655]: The `subscriptionReference` should be stable for the entire exchange.

Comment (nt-gt) [reply booking/src/main/java/org/dcsa/conformance/standards/booking/party/BookingCarrier.java:655]: (as in, for all notifications in the scenario)

File: booking/src/main/java/org/dcsa/conformance/standards/booking/checks/BookingChecks.java (hunk 1 above)
Comment (preetamnpr) [reply booking/src/main/java/org/dcsa/conformance/standards/booking/checks/BookingChecks.java:330]: Copy paste error.

**Review by nt-gt — APPROVED** [review]

**Pull request merged**